src/msg/async/frames_v2.h

```diff
@@ -169,18 +174,22 @@ static constexpr uint32_t FRAME_PREAMBLE_WITH_INLINE_SIZE =
 #define FRAME_LATE_STATUS_RESERVED_FALSE  0xe0
 #define FRAME_LATE_STATUS_RESERVED_MASK   0xf0
 
+#define FRAME_EARLY_DATA_COMPRESSED       0X1
```

Comment:
Why `..._EARLY_...`?

Reply:
The compression flag is sent as part of the preamble (which contains metadata about incoming size of next segments) - unlike the rest of the flags which are sent as part of the epilogue 
We can loose the EARLY and just call it `FRAME_DATA_COMPRESSED` if you think it's indicative enough

Reply:
Oh, I see; early is fine then. Maybe just put the above explanation as a comment in the code.